### PR TITLE
SPCTR-2921 Countdown - Hide and Redirect - Expiry options should work without manual refresh

### PR DESCRIPTION
**Pull request by aapatre**
### Description
The hide and redirect options initially only worked on page refresh, not providing a real time experience. Hence this has been improved.

- [JIRA](https://brainstormforce.atlassian.net/browse/SPCTR-2921)

### Types of changes

- UX Improvement

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/uagb-countdown.js [hunk 1]
```diff
@@ -212,6 +221,11 @@ UAGBCountdown = {
 		// If it's overtime, stop updating the markup and clear the interval.
 		if ( isOvertime ) {
 			clearInterval( this.countdownInterval[ mainSelector ] );
+
+			// Set flag variable to true, for usage in Countdown Pro code (like a signal).
+			if( data.timerEndAction === 'redirect' || data.timerEndAction === 'hide' && data.isFrontend ) {
```
Comment (rahulvarma722) [inline assets/js/uagb-countdown.js:226]: Yoda condition
```suggestion
			if( 'redirect' === data.timerEndAction || 'hide' === data.timerEndAction && data.isFrontend ) {
```

**Review by bsf-zanev — APPROVED** [review]
Looks good to me, just resolve the comment above.
Yoda condition, we must use.

File: assets/js/uagb-countdown.js (hunk 1 above)
Comment (aapatre) [reply assets/js/uagb-countdown.js]: Done

File: includes/blocks/countdown/attributes.php [hunk 2]
```diff
@@ -17,6 +17,7 @@
 return array_merge(
 	array(
 		'block_id'                     => '',
+		'isFrontend'                   => true,
```
Comment (adi3890) [inline includes/blocks/countdown/attributes.php:20]: @aapatre just wanted to confirm is this attribute already available in js.

**Review by AbhijitNage123 — APPROVED** [review]

File: includes/blocks/countdown/attributes.php (hunk 2 above)
Comment (aapatre) [reply includes/blocks/countdown/attributes.php:20]: This is meant to be an attribute exclusively for frontend (as we don't need to trigger a specific countdown condition in the editor).

**Review by adi3890 — APPROVED** [review]

**Pull request merged**